samples/scripts/storage.fsx

```diff
@@ -7,6 +7,12 @@ let myStorage = storageAccount {
     name "myfarmerstorage"
     sku Storage.Sku.Standard_LRS
     add_queues [ "queue1"; "queue2" ]
+    add_queues_with_same_metadata 
+        ["queue3"; "queue4"]
+        (Map [
```

Comment:
Do we need the `Map` function call? I would have thought that this can happen in the implementation of the keyword.

Reply:
I didn't really like having the Map there myself but I wasn't sure what the DSL should look like instead. Do you have any recommendations?

Is it possible to just use:
{
    "key" = "value"
}

I didn't think that an anonymous record would work because all keys are chosen by the user, so no specific structure.